dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -25,4 +25,9 @@ maxFieldLength=Maximum field length
 maxFieldLength.desc=Specifies maximum number of characters for the message field associated with the events for various sources. If you specify a value of 0 or -1, the maximum number of characters is unlimited.
 
 maxEvents=Maximum events per second
-maxEvents.desc=Specifies the maximum number of events to be sent per second. If you specify a value greater than zero, then it will be used for throttling the events.
\ No newline at end of file
+maxEvents.desc=Specifies the maximum number of events to be sent per second. If you specify a value greater than zero, then it will be used for throttling the events.
```

Comment:
Suggested changes to shorten the description:
> Specifies the maximum number of events to send per second. To throttle the events, specify a value greater than zero.

> Specifies the maximum number of events to send per second. A value greater than zero throttles the events.

Reply:
This message has already been GAed. If you think this message will have a significant impact to the customer, I can make the change - but otherwise, I think we can leave it as it is.